README.md

```diff
@@ -52,6 +52,7 @@ we support!
 | [`@carbon/colors`](./packages/colors)         | Work with IBM Design Language colors                                                                                                                |
 | [`@carbon/grid`](./packages/grid)             | Build layouts using the new 16 column grid system                                                                                                   |
 | [`@carbon/icons`](./packages/icons)           | Iconography assets. We also offer support in: [React](./packages/icons-react), [Angular](./packages/icons-angular), and [Vue](./packages/icons-vue) |
+| [`@carbon/pictograms`](.packages/pictograms)  | Pictogram assets |
```

Comment:
Would we want to mirror the tagline from above for React?

Reply:
That's a good call. Do we offer pictograms in Angular/Vue yet?

Reply:
@connor-leech not that I'm aware of!

Reply:
👍